src/OmniSharp.Roslyn.CSharp/Services/Types/TypeLookup.cs

```diff
@@ -45,7 +46,9 @@ public TypeLookupService(OmniSharpWorkspace workspace, FormattingOptions formatt
 
                     if (request.IncludeDocumentation)
                     {
-                        response.Documentation = DocumentationConverter.ConvertDocumentation(symbol.GetDocumentationCommentXml(), _formattingOptions.NewLine);
+                        string newLine = Environment.NewLine + Environment.NewLine;
+                        //VS Code renders a single new line for two newline sequences in the response, hence two new lines are passed in the lineEnding parameter 
```

Comment:
If this is a VS Code-specific tweak, why does it belong in omnisharp-roslyn? Shouldn't the fix be in omnisharp-vscode? After all, other editors use omnisharp-roslyn (e.g. Emacs, Vim, Atom, Sublime, etc.). Wouldn't this be a breaking change for them?